models/stg_zendesk__schedule_holiday.sql

```diff
@@ -0,0 +1,36 @@
+--To disable this model, set the using_schedules variable within your dbt_project.yml file to False.
+{{ config(enabled=var('using_schedules', True)) }}
+
+with base as (
+
+    select * 
+    from {{ ref('stg_zendesk__schedule_holiday_tmp') }}
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_zendesk__schedule_holiday_tmp')),
+                staging_columns=get_schedule_holiday_columns()
+            )
+        }}
+    from base
+),
+
+final as (
+    
+    select
+        _fivetran_deleted,
+        _fivetran_synced,
```

Comment:
should we cast this as a timestamp as well? or only platform-originated timestamps?

Reply:
bump

Reply:
I don't believe they are needed to be cast as these should always be in UTC timestamp format as they are fivetran generated. That being said, it doesn't hurt to add the cast.